books_service/models.py

```diff
@@ -0,0 +1,41 @@
+from django.db import models
+
+
+class Author(models.Model):
+    first_name = models.CharField(max_length=255)
+    last_name = models.CharField(max_length=255)
+
+    class Meta:
+        ordering = ["last_name", "first_name"]
+
+    @property
+    def full_name(self) -> str:
+        return str(self)
+
+    @property
+    def count_books(self) -> int:
+        return Book.objects.filter(author=self).count()
+
+    def __str__(self) -> str:
+        return f"{self.first_name} {self.last_name}"
+
+
+class Book(models.Model):
+    COVER_CHOICES = (
+        ("hard", "Hard"),
```

Comment:
i'm not sure, do we need duplicate this choises with differect characters?

Reply:
Not a duplication, read [here](https://docs.djangoproject.com/en/4.2/ref/models/fields/#choices)